convert2rhel/actions/system_checks/convert2rhel_latest.py

```diff
@@ -146,13 +147,35 @@ def run(self):
                 latest_available_version = (package_version[1], package_version[2], package_version[3])
 
         logger.debug("Found %s to be latest available version" % (latest_available_version[1]))
+
+        # Get source files that we’re running with import convert2rhel ; convert2rhel.__file__
+        running_convert2rhel_init_file = convert2rhel_file
+
+        # Run `rpm -qf <source file>` to get the installed convert2rhel package NEVRA
+        running_convert2rhel_NEVRA, return_code = utils.run_subprocess("rpm", "-qf", running_convert2rhel_init_file)
+
+        # Run `rpm -V <convert2rhel pkg NEVRA>` to make sure the user hasn’t installed a different convert2rhel version on top of a previously installed rpm package through other means than rpm (e.g. pip install from GitHub)
+        correct_rpm_convert2rhel_version, return_code1 = utils.run_subprocess("rpm", "-V", running_convert2rhel_NEVRA)
+
+        # If any of those steps fail, we print a warning that we could not determine the rpm release and use convert2rhel.{}version{} to compare with the latest packaged version
+        if return_code != 0 or return_code1 != 0:
+            logger.warning(
+                "Couldn't determine the rpm release, so the conversion will use this version of convert2RHEL. %s"
+                % system_running_convert2rhel_version
+            )
+            running_convert2rhel_version = system_running_convert2rhel_version
```

Comment:
I would keep the `running_convert2rhel_version` for the source version that comes from `from convert2rhel import __version__ as system_running_convert2rhel_version` and make a new variable name to represent the version that we've chosen as the right one to work with for the rest of the function.  For instance, `correct_convert2rhel_version`
```suggestion
            correct_convert2rhel_version = system_running_convert2rhel_version
```

Reply:
This is done!